cloud/pkg/controller/config/buffer.go

```diff
@@ -46,4 +55,26 @@ func init() {
 		QuerySecretBuffer = qsb
 	}
 	log.LOGGER.Infof("query secret buffer: %d", QuerySecretBuffer)
+
+	if peb, err := config.CONFIG.GetValue("pod-event-buffer").ToInt(); err != nil {
```

Comment:
Can we refer to PR #490 and add these configurations under buffer for both controller and deviceController ?
Lets wait till pr #490 is approved though

Reply:
ok

Reply:
@kadisi , the PR is merged. Please take a look and add configurations under buffer

Reply:
ok